dashboard/test/controllers/reference_guides_controller_test.rb

```diff
@@ -75,6 +75,28 @@ class ReferenceGuidesControllerTest < ActionController::TestCase
     assert_equal 'new content', editable_reference_guide.content
   end
 
+  test 'updating parent_reference_guide_key works' do
+    editable_reference_guide = create :reference_guide, course_version: @unit_group.course_version, parent_reference_guide_key: nil
+
+    sign_in @levelbuilder
+
+    assert_not_equal editable_reference_guide.content, 'new content'
+    File.expects(:write).with {|filename, _| filename.to_s.end_with? "#{editable_reference_guide.key}.json"}.once
+
+    old_last_position = ReferenceGuide.where(course_version_id: @unit_group.course_version.id, parent_reference_guide_key: @reference_guide.key).last.position
+
+    post :update, params: {
+      course_course_name: editable_reference_guide.course_offering_version,
+      key: editable_reference_guide.key,
+      parent_reference_guide_key: @reference_guide.key
+    }
+    assert_response :ok
+
+    editable_reference_guide.reload
+    assert_equal @reference_guide.key, editable_reference_guide.parent_reference_guide_key
+    assert editable_reference_guide.position > old_last_position
```

Comment:
Wondering about why you chose to check if it was greater instead of checking if it was equal to the old last position + 1

Reply:
for it to be correct, it just needs to be the last thing in the list and a unique value. in this specific case, it doesn't really matter too much, but for tests my approach is to usually have it just testing against the requirements, that way the tests are less brittle with future changes (i.e. if we needed to always increment by 2 for whatever reason in the future, this would still pass and the functionality it is testing would still work as expected)